tdrs-backend/README.md

```diff
@@ -167,33 +157,31 @@ Space (enter to skip): 1
 Targeted space <SPACE-1>.
 ```
 
-3.) Push the image to Cloud.gov (you will need to be in the same directory as`tdrs-backend/manifest.yml`):
-
-( **The `--var` parameter ingests a value into the ``((docker-frontend))`` environment variable in the manifest.yml**)
+2.) Push the image to Cloud.gov (you will need to be in the same directory as`tdrs-backend/manifest.yml`):
 
 ```bash
- $ cf push tdp-backend -f manifest.yml --var docker-backend=lfrohlich/tdp-backend:local
+ $ cf push tdp-backend -f manifest.yml
```

Comment:
leaving this note for the record: per @jtwillis92 currently, these steps for manual deploy are out of sync until we replace the contents of manifest.yml w/ the contents of the buildpack manifest. this can be addressed in #1035